frontend/src/views/CellGuide/components/common/FullScreenProvider.tsx

```diff
@@ -25,7 +31,7 @@ const FullScreenProvider: React.FC<React.PropsWithChildren<{}>> = ({
   });
 
   const enableFullScreen = () => setIsFullScreen(true);
-  const disableFullScreen = () => setIsFullScreen(false);
+  const disableFullScreen = useCallback(() => setIsFullScreen(false), []);
```

Comment:
wrap this in useCallback, since linter was complaining that re-creating this function every time was causing its dependent hooks to re-run every render